loglist2/logfilter.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
nit: this is (c) 2018 (it's mostly a copy of `../loglist/logfilter.go`, which was created in 2018)

Reply:
Done